src/config/mod.rs

```diff
@@ -1367,15 +1473,27 @@ impl Default for RaftDefaultCfConfig {
             prepopulate_block_cache: PrepopulateBlockCache::Disabled,
             format_version: 2,
             checksum: ChecksumType::CRC32c,
+            max_compactions: 0,
             titan: TitanCfConfig::default(),
         }
     }
 }
 
 impl RaftDefaultCfConfig {
     pub fn build_opt(&self, cache: &Cache) -> RocksCfOptions {
+        let limiter = if self.max_compactions > 0 {
```

Comment:
I suggest leave raft db untouched as it's recommended to use raft log engine.

Reply:
The code for all cfs is shared except for build_opt. Might as well leave it even though it's mostly useless even when raft-engine is disabled.